app/scripts/views/mixins/resend-mixin.js

```diff
@@ -23,12 +28,10 @@ define([
       // The button is hidden after the forth click for 5 minutes, then
       // start the process again.
 
-      this._attemptedSubmits++;
-
+      this._emailResend.requestResend();
```

Comment:
Maybe rename `requestResend` to `incrementRequestCount` or something along those lines. `requestResend` implies the the called function does the resend request.